src/Microsoft.DotNet.Build.Tasks.Feed/BlobUrlInfo.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.WindowsAzure.Storage.Auth;
+using System.Text.RegularExpressions;
+
+namespace Microsoft.DotNet.Build.Tasks.Feed
+{
+    class BlobUrlInfo
+    {
+        private const string accountNameRegex = @"(?<accountname>[a-z0-9]+)\.blob\.core\.windows\.net.*";
```

Comment:
Why bake in `blob.core.windows.net`? I think I would make that a little more generic.

Reply:
Will fix.